rust/migrate-wicked/tests/wireless_eap/wicked_xml/eap.xml

```diff
@@ -0,0 +1,116 @@
+<interface origin="compat:suse:/etc/sysconfig/network/ifcfg-wlan_eap0">
+  <name>wlan_eap0</name>
+  <control>
+    <mode>manual</mode>
+  </control>
+  <firewall/>
+  <wireless>
+    <ap-scan>1</ap-scan>
+    <networks>
+      <network>
+        <essid>test</essid>
+        <scan-ssid>true</scan-ssid>
+        <mode>ap</mode>
+        <access-point>12:34:56:78:9a:bc</access-point>
+        <key-management>wpa-eap</key-management>
+        <wpa-eap>
+          <method>tls</method>
+          <auth-proto>wpa,rsn</auth-proto>
+          <pairwise-cipher>TKIP</pairwise-cipher>
+          <group-cipher>TKIP,WEP40</group-cipher>
```

Comment:
This is a very uncommon configuration value! Of course for testing it's ok. And it should not make any problem in the configuration later on, because it's just a list of allowed ciphers.
Just saying, that not someone is using these test config as valid examples later on!